apps/i18n/luna/en.json

```diff
@@ -104,7 +104,7 @@
     "Log out": "Log out",
     "Login reminder": "Login reminder",
     "Login review approved": "Login review has been approved, connecting assets...",
-    "LoginExpireMsg": "Login session expired, please re-login",
+    "LoginExpireMsg": "The current login has timed out, please do not close the window. Log in in a new window to restore the current page. If not, please refresh the page.",
```

Comment:
Your session has timed out. Please do not close this window. Log in through a new window to restore your current page. If this does not resolve the issue, please refresh the page.